h/api/presenters.py

```diff
@@ -54,7 +54,8 @@ def tags(self):
 
     @property
     def target(self):
-        target = {'source': self.annotation.target_uri}
+        target = {'source': self.annotation.target_uri,
+                  'scope': self.annotation.target_uri_normalized}
```

Comment:
This issue <https://github.com/hypothesis/h/issues/3261> that I ran into elsewhere was also to do with unwanted stuff getting into the search index because we use the presenters for it. I was thinking we needed a separate presenter for the search index though, not that we didn't want to use a presenter for the search index at all?